charts/internal/calico/templates/node/vpa.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.vpa.enabled }}
```

Comment:
Please note `updateMode` in this resource. It should probably have a different value.

Furthermore, the cluster-proportional autoscaler entities should not be enabled if `.Values.vpa.node` is set.